managed/src/main/java/com/yugabyte/yw/controllers/TabletServerController.java

```diff
@@ -52,7 +53,7 @@ public Result list() {
                 tabletServers.add(tabletServer.getHost());
               });
     } catch (Exception e) {
```

Comment:
We can not remove this try catch , right?

Reply:
You can have try with only finally. 
So remove this.

Reply:
we need to catch this as `client.listTabletServers();` throwing an exception .

Reply:
or I can change the method signature